src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/settings/SettingsMainWindow.java

```diff
@@ -153,8 +153,8 @@ private void showSettingsClosedWarning() {
         alert.setContentText(translator.getString("monitoring.settings.closewarning.text"));
         Stage stage = (Stage) alert.getDialogPane().getScene().getWindow();
         stage.getIcons().add(new Image("/icon.png"));
+        stage.setAlwaysOnTop(true);
         alert.initModality(Modality.APPLICATION_MODAL);
-        alert.initOwner(window);
```

Comment:
Das gefällt mir. Sollte man vielleicht einfach bei allen machen, oder? Irgendwie gibt es trotz meinem PR immernoch Situationen, in denen man das Hauptfenster auswählen kann, ohne dass der Dialog aktiv wird...